### ISSUE TITLE
Add "File -> New Container window" feature

### ISSUE THREAD

**Issue by bravecrayon** [mozilla/multi-account-containers#2449]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

1. Click File menu
2. There is no "New Container window" option

### Actual behavior

There is no ability to create a new window with tab in a specific container.

### Expected behavior

Expected to find "New Container window" in the File menu with a fly-out menu for choosing a container. Upon selecting a container from this fly-out menu, a new window is created with a tab in the selected container.

### Additional informations

This is a feature request. At first I followed the link to the [Mozilla Connect](https://connect.mozilla.org/t5/forums/postpage/board-id/ideas) but it didn't seem specific to this plugin. Please advise of a proper way to ask for features.

**Comment by dannycolin** [comment]
> At first I followed the link to the [Mozilla Connect](https://connect.mozilla.org/t5/forums/postpage/board-id/ideas) but it didn't seem specific to this plugin. Please advise of a proper way to ask for features.

Since the menubar is rendered by Firefox itself, it'd be best to file a bug report at https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Menus

**Comment by bravecrayon** [comment]
@dannycolin I might not have been clear. I believe the Multi-site Containers extension adds a menu item for "New Container tab" and my feature request is for adding a "New Container window" menu item that functions similiarly.
![CleanShot 2022-11-21 at 08 56 10@2x](https://user-images.githubusercontent.com/43159603/203073509-f25edfa4-47d1-4362-9e55-fa10ce8e46de.png)

**Comment by dannycolin** [comment]
That's exactly the menu I'm talking about. Lots of people are confused by this but Containers are a built-in feature in Firefox and Multi-Account Containers only enhances it. However, it can't change menus like this one.

**Comment by bravecrayon** [comment]
Yeah, I guess that is confusing. I *thought* "New Container tab" started showing up only after I installed the extension, but if I uninstall it that menu item remains.
